Makefile

```diff
@@ -71,27 +71,35 @@ manifests: trusted-cluster-gen
 		-register-server-image $(REG_SERVER_IMAGE)
 
 cluster-up:
-	scripts/create-cluster-kind.sh
+	RUNTIME=$(RUNTIME) scripts/create-cluster-kind.sh
 
 cluster-cleanup:
-	$(KUBECTL) delete -f manifests/trusted_execution_cluster_cr.yaml
-	$(KUBECTL) delete -f manifests/trusted_execution_cluster_crd.yaml
-	$(KUBECTL) delete -f manifests/operator.yaml
+	$(KUBECTL) delete -f $(DEPLOY_PATH)/trusted_execution_cluster_cr.yaml
+	$(KUBECTL) delete -f $(CRD_YAML_PATH)/trusted-execution-clusters.io_trustedexecutionclusters.yaml
+	$(KUBECTL) delete -f $(DEPLOY_PATH)/operator.yaml
 
 
 cluster-down:
-	scripts/delete-cluster-kind.sh
+	RUNTIME=$(RUNTIME) scripts/delete-cluster-kind.sh
+
+CONTAINER_CLI ?= podman
+RUNTIME ?= podman
 
 image:
-	podman build --build-arg build_type=$(BUILD_TYPE) -t $(OPERATOR_IMAGE) -f Containerfile .
-	podman build --build-arg build_type=$(BUILD_TYPE) -t $(COMPUTE_PCRS_IMAGE) -f compute-pcrs/Containerfile .
-	podman build --build-arg build_type=$(BUILD_TYPE) -t $(REG_SERVER_IMAGE) -f register-server/Containerfile .
+	$(CONTAINER_CLI) build --build-arg build_type=$(BUILD_TYPE) -t $(OPERATOR_IMAGE) -f Containerfile .
+	$(CONTAINER_CLI) build --build-arg build_type=$(BUILD_TYPE) -t $(COMPUTE_PCRS_IMAGE) -f compute-pcrs/Containerfile .
+	$(CONTAINER_CLI) build --build-arg build_type=$(BUILD_TYPE) -t $(REG_SERVER_IMAGE) -f register-server/Containerfile .
+
+ifeq ($(CONTAINER_CLI),podman)
+TLS_VERIFY_FLAG = --tls-verify=false
```

Comment:
We actually just add a new variable called `PUSH_FLAGS` can you please adjust this to use that variable